lookout/style/typos/preprocessing.py

```diff
@@ -0,0 +1,32 @@
+from typing import Set, Union
+
+import pandas
+
+from lookout.style.typos.utils import Columns
+
+
+def check_split(text: str, tokens: Set[str]) -> bool:
+    """
+    Check whether all tokens of the text belong to the given set.
+
+    :param text: String of space-separated tokens.
+    :param tokens: Set of tokens (reference vocabulary).
+    :return: True when all tokens of the `text` belong to `tokens`.
+    """
+    for token in text.split():
+        if token not in tokens:
+            return False
+    return True
+
+
+def filter_splits(data: Union[pandas.DataFrame, str], tokens: Set[str]) -> pandas.DataFrame:
+    """
+    Leave rows in a dataframe whose splits' tokens all belong to some vocabulary.
+
+    :param data: Dataframe or its .csv dump, containing column Columns.Split.
+    :param tokens: Set of tokens (reference vocabulary).
+    :return: Filtered dataframe.
+    """
+    if isinstance(data, str):
```

Comment:
As @vmarkovtsev mentioned in another PR, it's important to split the IO part and the logic part (to have the biggest possible part of the codebase made of [pure functions](https://en.wikipedia.org/wiki/Pure_function)). This enhances testability and maintainability by a lot.

We should therefore remove this check.

Reply:
Very well, will do that everywhere.